src/components/portrait/portrait.style.js

```diff
@@ -0,0 +1,246 @@
+
+import BaseTheme from '../../style/themes/base';
+import Icon from '../icon';
+import PortraitGravatar from './portrait-gravatar.component';
+import PortraitInitials from './portrait-initials.component';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import { isClassic } from '../../utils/helpers/style-helper';
+
+/* eslint-disable key-spacing, no-multi-spaces */
+
+const sizeParams = {
+  'extra-small':  { dimensions: 24,  iconDimensions: 16 },
+  small:          { dimensions: 32,  iconDimensions: 16 },
+  'medium-small': { dimensions: 40,  iconDimensions: 24 },
+  medium:         { dimensions: 56,  iconDimensions: 32 },
+  'medium-large': { dimensions: 72,  iconDimensions: 32 },
+  large:          { dimensions: 104, iconDimensions: 48 },
+  'extra-large':  { dimensions: 128, iconDimensions: 64 }
+};
+
+const sizeParamsClassic = {
+  'extra-small':  { dimensions: 25,  iconDimensions: null, iconPadding: 4  },
+  small:          { dimensions: 30,  iconDimensions: null, iconPadding: 6  },
+  'medium-small': { dimensions: 40,  iconDimensions: 24,   iconPadding: 12 },
+  medium:         { dimensions: 60,  iconDimensions: 32,   iconPadding: 14 },
+  'medium-large': { dimensions: 70,  iconDimensions: 32,   iconPadding: 18 },
+  large:          { dimensions: 100, iconDimensions: 48,   iconPadding: 25 },
+  'extra-large':  { dimensions: 120, iconDimensions: 64,   iconPadding: 27 }
+};
+
+/* eslint-enable key-spacing, no-multi-spaces */
+
+function stylingForBorder(props) {
+  if (isClassic(props.theme) || props.darkBackground || props.showGravatar || props.showCustomImg) {
+    return css``;
+  }
+
+  if (props.showIcon) {
+    return css`border: 1px dashed ${props.theme.portrait.border};`;
+  }
+
+  // Border for initials
+  return css`border: 1px solid ${props.theme.portrait.border};`;
+}
+
+function stylingForSize({ size, theme }) {
+  const params = (isClassic(theme) ? sizeParamsClassic[size] : sizeParams[size]);
+
+  if (!params) {
+    return css``;
+  }
+
+  return css`
+    width:  ${params.dimensions}px;
+    height: ${params.dimensions}px;
+  `;
+}
+
+function stylingForShape({ shape }) {
+  switch (shape) {
+    case 'standard': return css`border-radius: 0px;`;
+    case 'circle': return css`border-radius: 50%;`;
+    case 'leaf': return css`border-radius: 10% 40% 10%;`;
+    default: return css``;
+  }
+}
+
+function stylingForIcon({ size, theme, darkBackground }) {
+  const isThemeClassic = isClassic(theme);
+  const params = (isThemeClassic ? sizeParamsClassic[size] : sizeParams[size]);
+
+  if (!params) {
+    return css``;
+  }
+
+  let color = theme.portrait.border;
+  let backgroundColor = theme.portrait.background;
+  let iconPadding = (params.dimensions - params.iconDimensions) / 2;
+
+  if (darkBackground) {
+    color = theme.portrait.background;
+    backgroundColor = theme.portrait.border;
+  }
+
+  if (isThemeClassic) {
+    color = (darkBackground ? '#ffffff' : '#335c6d');
```

Comment:
If the hardcoded values are in DLS version of the components we need to bring them from themes

Reply:
These hardcoded values are for the Classic theme, not DLS (they're within [the `isThemeClassic` block](https://github.com/Sage/carbon/blob/dd2fab1/src/components/portrait/portrait.style.js#L88)).